app/lib/utils/translations/translations_home.dart

```diff
@@ -156,6 +159,13 @@ class TranslationsEnHome implements TranslationsHome {
   String get unknownError => 'Unknown Error';
   @override
   String get openMapApplication => 'Open Map Application';
+  @override
+  String get unRegister => 'Unregister';
+  @override
+  String get unRegisterDescriptoin =>
+      'If you want to cancel your registration for the meeting, you can cancel it here.';
+  @override
+  String get unRegisterDialogTitle => 'Make sure you want to cancel the registration';
```

Comment:
I think we can keep this more simple:
```suggestion
  String get unRegisterDialogTitle => 'Unregister from the next cycle?';
```